sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,121 @@
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_resetWavefrontCorrection</EFDB_Topic>
+    <EFDB_Topic>MTAOS_command_resetCorrection</EFDB_Topic>
     <Description>Reset the current wavefront error correction. The aggregated degree of freedom in optical feedback control will be reset to the initial state.</Description>
-    <item>
-      <EFDB_Name>value</EFDB_Name>
-      <Description>This value is ignored.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_issueWavefrontCorrection</EFDB_Topic>
+    <EFDB_Topic>MTAOS_command_issueCorrection</EFDB_Topic>
     <Description>Issue the wavefront correction to subsystems based on the most recently measured wavefront error.</Description>
-    <item>
-      <EFDB_Name>value</EFDB_Name>
-      <Description>This value is ignored.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processCalibrationProducts</EFDB_Topic>
-    <Description>Process new calibration products.</Description>
+    <EFDB_Topic>MTAOS_command_selectSources</EFDB_Topic>
+    <Description>Run source selection algorithm for a specific field and visit configuration.</Description>
     <item>
-      <EFDB_Name>directoryPath</EFDB_Name>
-      <Description>Absolute directory path to calibration products.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>ra</EFDB_Name>
+      <Description>Right Ascension of the field in ICRS coordinates.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>hourangle</Units>
```

Comment:
Why the `ra` use the `hourangle` and `decl` use the `degree`? Why not use the same unit?

Reply:
Why we could use the same unit of `hourangle` or `degree` for both of them?

Reply:
that's how astronomers use RA and declination...